resources/views/admin/addProduct.blade.php

```diff
@@ -0,0 +1,147 @@
+@extends('layouts.base')
+@push('scripts')
+    <script src="{{ asset('js/manage-products/product.js') }}"></script>
```

Comment:
Moet deze niet ook defer hebben?
Reden: Lokaal gaat alles heel snel, live gaat het traag het kan dat fouten opleveren als je verwacht dat iets er is.

Reply:
Dit document heeft alleen functies en zelf geen script. Script onderdelen worden aangeroepen in de code en dus is defer niet gewenst